### ISSUE TITLE
Add Hebrew translation in crowdin

### ISSUE THREAD

**Issue by Y-PLONI** [huynhsontung/Screenbox#418]
Could you please add in crowdin an option to translate the app into Hebrew?
Thank you very much!
Your app is very beautiful!
I'm considering removing windows media player entirely!

**Comment by huynhsontung** [comment]
Thank you for your kind words! Hebrew has been added.

**Comment by Y-PLONI** [comment]
I translated a significant part of the software, I would be happy to check if there are any bugs due to the combination between Hebrew and English. You will be able to release a new version? [you can also release software with the same version number, so that the software is not updated for everyone, and only those who remove and install receive the update].

**Comment by Y-PLONI** [comment]
> I translated

I'm done, I can reach 100% only after I see the translation inside the software.
thanks!

**Comment by huynhsontung** [comment]
@Y-PLONI I have added some screenshots to help with the context. It should be good enough for a release as is. Thank you!

**Comment by iLiranS** [comment]
Hi, the Hebrew version is broken, the videos are vertically flipped, the nav bar is broken as well (I guess it's about the RTL behavior). If possible, I would like an option to choose preferred language as well 😅

**Comment by Y-PLONI** [comment]
@iLiranS Right, I just saw that and will start another issue on it.
In my opinion, it is better to configure the software as if it were in English, with a Hebrew language file [most problems are caused by the definition that the application is in Hebrew, and not by the fact that the text displayed on the screen is in Hebrew]. I don't know if it's possible...

**Comment by huynhsontung** [comment]
Hebrew translation has been added in #424 and subsequently #434
